client/app/scripts/reducers/root.js

```diff
@@ -84,6 +84,7 @@ export const initialState = makeMap({
   viewport: makeMap(),
   websocketClosed: false,
   zoomCache: makeMap(),
+  serviceImages: makeMap({ fetching: false })
```

Comment:
1. I don't think we will necessarily keep everything Weave Cloud related in a single key. It's possible we will read/update existing things all over the Scope state tree.

2. A user would only ever see the one details panel at a time, and therefore only one loading state is necessary, but I agree that would be more correct; I will fix that.